src/identity/components/Address.tsx

```diff
@@ -0,0 +1,19 @@
+import { useIdentityContext } from '../context';
+import { getSlicedAddress } from '../getSlicedAddress';
+import { cn, text } from '../../styles/theme';
+import type { AddressReact } from '../types';
+
+export function Address({ address = null, className }: AddressReact) {
+  const { address: contextAddress } = useIdentityContext();
+  if (!contextAddress && !address) {
+    throw new Error(
+      'Address: an Ethereum address must be provided to the Identity or Address component.',
+    );
+  }
+
+  return (
+    <span className={cn(text.label2, className)}>
+      {getSlicedAddress(contextAddress ?? address)}
```

Comment:
nit. for later, we might need an hook for this, so it does not cause a new instance of this function at every render.